### DISCUSSION TITLE
Enkaz altında canlılık haritası

### DISCUSSION THREAD

**Discussion by halilburakyilmaz** [acikkaynak/deprem-yardim-projesi#103] [Ideas]
Bu deprem değil fakat bundan sonraki depremlerde telefon ve akıllı saatleri kullanarak enkaz altı bildirimi atabilecek bir uygulama yapılabileceğini düşünüyorum.

Deprem anında ilgili bölgede olup telefonunda bu uygulama yüklü olanlar için konum ve hareket-ses bildirimi otomatik aktifleşecek. şarjı verimli kullanmak adına belirli periyotlar ile uygulama konum ve hareket/ses bilgisini otomatik paylaşacak. 

Eğer 
1. konum sabit ya da önceki bildirim ile çok yakın ve hareket/insan sesi yok ise cansızlık ya da sahibinden uzak telefon demek olacak.
2. konum sabit fakat hareket ya da ses algısı var ise enkaz altı canlılık bildirimi olacak.
3. Telefon sahibinde ise hayattayım ve sağlıklıyım bilgisini kendisi bildirecek.

Şarj tüketimini sınırlandırmak adına periyod aralıkları giderek uzatılabilir.
Telefonu elinde olmayanlar için yine belirli aralıklarla uygulama hoparlörden etraftakilerin ses çıkarmasını ister. Bu sayede depremin ilk anından itibaren enkaz altı canlılık haritası otomatik olarak oluşur ve önceliklendirme yapılabilinir.

**Comment by sercangoger** [comment]
mobil fikirlerinizi lütfen burada belirtiniz.

https://github.com/acikkaynak/afetharita-roadmap/discussions/9
